my-app/src/Components/WeatherAlerts/WeatherAlerts.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default function WeatherAlerts(props) {
+  return (
+    <>
```

Comment:
` <>` empty tags are not needed if you have a common tag that wraps all the markup of the component